pkg/asset/ignition/bootstrap/baremetal/template.go

```diff
@@ -111,14 +117,40 @@ func GetTemplateData(config *baremetal.Platform, networks []types.MachineNetwork
 	templateData.ExternalStaticDNS = config.BootstrapExternalStaticDNS
 	templateData.ExternalMACAddress = config.ExternalMACAddress
 
-	_, externalURLv6 := externalURLs(config.APIVIPs)
+	// If the user has manually set disableVirtualMediaTLS to False in the Provisioning CR,
+	// then enable TLS in ironic
+	templateData.DisableIronicVirtualMediaTLS = true
+	protocol := "http"
+
+	provisioningCR := struct {
+		spec struct {
+			disableVirtualMediaTLS *bool `yaml:"disableVirtualMediaTLS"`
+		} `yaml:"spec"`
+	}{}
+	provisioningCRFile, err := os.Open("99_baremetal-provisioning-config.yaml") // which directory will this be in?
```

Comment:
It is in `ocp/ostest/saved-assets/openshift/99_baremetal-provisioning-config.yaml` on dev-scripts. I think `ocp/ostest` is provided as an argument, while the rest is hardcoded.

Reply:
I believe you can use the dependencies mechanism to resolve this (you'll need to update the function signature to pass the `dependencies` object). You'll need to `get` the dependency of type `&manifests.Openshift` (fortunately, it's already required by this asset), then you can iterate over its FileList to find the right file.

Reply:
Thanks, that worked wonderfully!